src/main/kotlin/frc/robot/subsystems/conveyor/ConveyorIOReal.kt

```diff
@@ -1,27 +1,54 @@
 package frc.robot.subsystems.conveyor
 
-import com.ctre.phoenix6.configs.Slot0Configs
+import com.ctre.phoenix6.configs.*
 import com.ctre.phoenix6.controls.VelocityVoltage
 import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix6.signals.InvertedValue
+import com.ctre.phoenix6.signals.NeutralModeValue
 import edu.wpi.first.units.Angle
 import edu.wpi.first.units.Measure
 import edu.wpi.first.units.Units
 import edu.wpi.first.units.Velocity
 import frc.robot.Ports
+import frc.robot.lib.LoggedTunableNumber
 
 class ConveyorIOReal : ConveyorIO {
     override val inputs = LoggedConveyorInputs()
     private val roller = TalonFX(Ports.Conveyor.MOTOR_ID)
     private val control = VelocityVoltage(0.0).withEnableFOC(true)
 
     init {
-        roller.configurator.apply(ConveyorConstants.MOTOR_CONFIG)
+        val config = TalonFXConfiguration()
+            .withMotorOutput(
```

Comment:
I think using `apply` here will be more concise

Reply:
c4f346c9dd1c91ccb5558df0b623b80f681b919a